server/mailet/mock-smtp-server/src/main/java/org/apache/james/mock/smtp/server/model/Response.java

```diff
@@ -121,15 +105,14 @@ public final boolean equals(Object o) {
         if (o instanceof Response) {
             Response response = (Response) o;
 
-            return Objects.equals(this.serverRejected, response.serverRejected)
-                && Objects.equals(this.code, response.code)
+            return Objects.equals(this.code, response.code)
                 && Objects.equals(this.message, response.message);
         }
         return false;
     }
 
     @Override
     public final int hashCode() {
-        return Objects.hash(code, message, serverRejected);
+        return Objects.hash(code, message);
```

Comment:
is there a test for that?

Reply:
Of course check on master, there's tests for equals/hashCode for all POJO in this project.